tietokanta/src/main/resources/db/migration/V1_923__.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE sopimus_tehtavamaara
+(
+    id        SERIAL PRIMARY KEY,
+    urakka    INTEGER NOT NULL REFERENCES urakka (id),
+    tehtava   INTEGER NOT NULL REFERENCES toimenpidekoodi (id),
+    maara     NUMERIC,
+    muokattu  TIMESTAMP,
+    muokkaaja INTEGER REFERENCES harja.public.kayttaja (id),
```

Comment:
harja.public osuuden voisi poistaa tältä riviltä ja lisätä
luotu              TIMESTAMP    NOT NULL DEFAULT NOW()

Reply:
Hyvä huomio, Idean autocorrect oli taas liian ahkera